uint8 TYPE_PRESSURE_RISE=0
uint8 TYPE_PRESSURE_DROP=1
uint8 type
